File: SwiftProAI.Core/aspnet-core/src/ThinknInsurTech.Core.Shared/Registration/Enum.cs

namespace ThinknInsurTech.Registration
{
    public enum EnumRegistrationStatus
    {
        UnderInvestigation = 1,
        Adjusters = 2,
        PendingInvoices = 3,
        CompletedInvoices = 4,
        Cancelled = 5
    }

    public enum EnumExpensesStatus
    {
        EXP001_PENDING_FOR_APPROVAL = 1,
        EXP002_PENDING_FOR_PAYMENT = 2,
        EXP003_PAYMENT_DONE = 3,
        EXP004_SUBMITTED = 4,
        EXP005_REJECTED = 5,
        EXP006_SUBMIT_WITHOUT_PAYMENT = 6
    }

}

File: SwiftProAI.Web/angular/src/app/shared/common/registration/enum.ts

export enum EnumRegistrationStatus {
    UnderInvestigation = 1,
    Adjusters = 2,
    PendingInvoices = 3,
    CompletedInvoices = 4,
    Cancelled = 5
}

export enum Enum3rdPartyRegistrationStatus {
    UnderInvestigation = 1,
    Completed = 4,
    Cancelled = 5
}

export enum EnumGroupType {
    Admin = 1,
    Adjuster = 2,
    Editor = 3,
    Finance = 4,
    Others = 5
}

export enum ExpensessStatusGroupType {
    PendingForApproval = 'EXP001',
    PendingForPayment = 'EXP002',
    PaymentDone = 'EXP003',
    Submitted = 'EXP004',
    Rejected = 'EXP005',
    SubmitWithoutPayment = 'EXP006'
}

export enum ExpensessClaimType {
    Expenses = 'ExClaimAppr001',
    Claims = 'ExClaimAppr002'
}

export enum ThirdPartyCaseViewApprovalStatus {
    Approved = 'Approved',
    Rejected = 'Rejected',
    PendingApproval = 'Pending Approval'
}

export enum CasePoliceReportSummaryType {
    Generated = 'Generated',
    Edited = 'Edited'
}